client/organization.go

```diff
@@ -17,8 +17,8 @@ type Organization struct {
 	CreatedAt                           string  `json:"createdAt"`
 	UpdatedAt                           string  `json:"updatedAt"`
 	Role                                string  `json:"role"`
-	IsSelfHosted                        bool    `json:"isSelfHosted"`
-	IsSelfHostedK8s                     bool    `json:"isSelfHostedK8s"`
+	IsSelfHosted                        bool    `json:"isSelfHosted" tfschema:"-"`
```

Comment:
IsSelfHosted no longer exists in API, it should be removed

Reply:
removed.